### PR TITLE
Preparing release 8.1.0

### PR DESCRIPTION
**Pull request by glasserc**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ ujson==1.35
 urllib3==1.22
 venusian==1.1.0
 waitress==1.1.0
-WebOb==1.7.4
+WebOb==1.8.0rc1
```
Comment (Natim) [inline requirements.txt]: Really?

**Comment by Natim** [comment]
Can we check the changelog according to https://github.com/Kinto/kinto/compare/8.0.0...master ?

File: requirements.txt (hunk 1 above)
Comment (glasserc) [reply requirements.txt]: Hmm, oops? I just did `make build-requirements` and committed without really reviewing it. I'll roll it back..

Comment (Natim) [reply requirements.txt]: No worries just checking :)

**Comment by leplatrem** [comment]
It should be 8.1.0 because we have new features.

And since #1439 was reviewed we could include it

**Pull request merged**